pandas/core/indexes/period.py

```diff
@@ -133,29 +133,43 @@ class PeriodIndex(DatelikeOps, DatetimeIndexOpsMixin, Int64Index):
     Parameters
     ----------
     data : array-like (1-dimensional), optional
-        Optional period-like data to construct index with
+        Optional period-like data to construct index with.
     copy : bool
-        Make a copy of input ndarray
+        Make a copy of input ndarray.
     freq : string or period object, optional
-        One of pandas period strings or corresponding objects
+        One of pandas period strings or corresponding objects.
     start : starting value, period-like, optional
         If data is None, used as the start point in generating regular
         period data.
     periods : int, optional, > 0
         Number of periods to generate, if generating index. Takes precedence
-        over end argument
+        over end argument.
     end : end value, period-like, optional
         If periods is none, generated index will extend to first conforming
-        period on or just past end argument
+        period on or just past end argument.
     year : int, array, or Series, default None
```

Comment:
general note: can you change `default None` in `optional` where relevant? (I think it are most of the keywords)